### PR TITLE
Storage home

### PR DESCRIPTION
**Pull request by ezequielc**
@alex-deaconu Please review.

https://apps-stage-0.risevision.com/storage

Thanks!

### CODE REVIEW

File: web/partials/storage/files-list.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        </div>
+      </td>
+      <td ng-if="!fileIsFolder(file)" class="col-sm-2 hidden-xs">
+        <span> {{file.name | fileTypeFilter}}</span>
+      </td>
+      <td ng-if="!fileIsFolder(file)" class="col-sm-2 visible-lg">
+        <span> {{file.size | fileSizeFilter}}</span>
+      </td>
+      <td ng-if="!fileIsFolder(file)" class="col-sm-2 visible-lg">
+        <span> {{file.updated | lastModifiedFilter}}</span>
```
Comment (alex-deaconu) [inline web/partials/storage/files-list.html]: @ezequielc I'm wondering if for consistency we should use the date filter that is used for other apps: https://github.com/Rise-Vision/rise-vision-apps/blob/master/web/partials/displays/displays-list.html#L58? Thanks.

Comment (ezequielc) [inline web/partials/storage/files-list.html]: @alex-deaconu Good point, it makes total sense. I will remove this filter and related code. Thanks

File: web/partials/storage/files-list.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+              <span ng-show="reverseSort">
+                <i class="fa fa-long-arrow-up"></i>
+              </span>
+              <span ng-show="!reverseSort">
+                <i class="fa fa-long-arrow-down"></i>
+              </span>
+            </span>
+          </a>
+        </th>
+        <th class="col-sm-2 visible-lg" ng-click="orderByAttribute = fileSizeOrderFunction; reverseSort = !reverseSort">
```
Comment (alex-deaconu) [inline web/partials/storage/files-list.html]: @ezequielc not sure why, but the sort by size doesn't work very well. Can you take a look? Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc couple of comments. Otherwise code looks good, thanks for the much needed refactoring!

File: web/partials/storage/files-list.html (hunk 2 above)
Comment (ezequielc) [inline web/partials/storage/files-list.html]: @alex-deaconu  I found the same issue in current Storage, but I managed to fix it. It needed a check in `fileSizeOrderFunction()`. Thanks

**Comment by ezequielc** [comment]
Thanks @alex-deaconu. I implemented both suggestions.  Please review. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc looks good. Thanks for fixing! :+1:

**Pull request merged**